.travis.yml

```diff
@@ -97,7 +115,34 @@ deploy:
       tags: true
       repo: coveo/search-ui
       all_branches: true
-      condition: "! $TRAVIS_TAG =~ .*beta.*"
+      condition: "$HAS_RELEASE_TAG == true"
+  - provider: s3
+    access_key_id: AKIAQ6FOLK5RDONGHVBY
+    secret_access_key:
+      secure: XL74wJ5088h082Ev6ZwPdEInliIF7/Dqvqz4rJYaXS3CU/KpsoeDhtV4wnF7Dr6P61AF6x8B4Q4uJKnOYBpg7oBjY7efsCDTJo2FeGxqM7j58Trdmh+kfDLmo9Rt06mTyvhKA/DC9tb7/Didnqs4/zDW0NUTqWu5ggnu0GO54kqoZon776ZCVrLzvSPpDh1VLqNTK5Z2ozIlO3XJc3y+QZeeG/yJBb3RcOnhv91+EIHib5y+HU5SoBIq71m2dbrR3q+V80mX+b9RAJDyb6wRjF8lDnrybaMF6Vf+FfzDj3fu1Z+juro8LN8afwMwKCZ4t8UHEB5n5/eD3oD6NwIkOQEPWMshbJr3UdM524dFrtcKQ5F7nvY8ZCE1a6zMGofmzINOb+qpiR5hlvIP375TaTVt7wIe7MM1CIuBferIqjL9fPmaMqcNRAxDt1HpI9HKOJ5ETSVXEpxviaBqYJjACwXgaXU4+5GsmcSprBSaBuBGJTVWQ5N7ewsxRP/2c3XmtMkB9lc+exwpF8/FvsJ6RbzNi1RhbA78r67XJcuK/2xD4aVYByd6PETVM1I7Bq0+LrECeEDqsTMcGHfAwc6ir69Lw0t/KVG/yHwbWyfwiYbkajfcQWYU669rD6ikY1B87assrqPDXneu65FxmoCYDMwBGRp8/w2gS7HSZmom5zA=
+    bucket: coveo-ndev-binaries
+    local-dir: bin
+    upload-dir: proda/StaticCDN/searchui/nightly
+    acl: public_read
+    skip_cleanup: true
+    on:
+      tags: true
+      repo: coveo/search-ui
+      all_branches: true
+      condition: "$IS_NIGHTLY == true"
+  - provider: s3
+    access_key_id: AKIAQ6FOLK5RDONGHVBY
+    secret_access_key:
+      secure: XL74wJ5088h082Ev6ZwPdEInliIF7/Dqvqz4rJYaXS3CU/KpsoeDhtV4wnF7Dr6P61AF6x8B4Q4uJKnOYBpg7oBjY7efsCDTJo2FeGxqM7j58Trdmh+kfDLmo9Rt06mTyvhKA/DC9tb7/Didnqs4/zDW0NUTqWu5ggnu0GO54kqoZon776ZCVrLzvSPpDh1VLqNTK5Z2ozIlO3XJc3y+QZeeG/yJBb3RcOnhv91+EIHib5y+HU5SoBIq71m2dbrR3q+V80mX+b9RAJDyb6wRjF8lDnrybaMF6Vf+FfzDj3fu1Z+juro8LN8afwMwKCZ4t8UHEB5n5/eD3oD6NwIkOQEPWMshbJr3UdM524dFrtcKQ5F7nvY8ZCE1a6zMGofmzINOb+qpiR5hlvIP375TaTVt7wIe7MM1CIuBferIqjL9fPmaMqcNRAxDt1HpI9HKOJ5ETSVXEpxviaBqYJjACwXgaXU4+5GsmcSprBSaBuBGJTVWQ5N7ewsxRP/2c3XmtMkB9lc+exwpF8/FvsJ6RbzNi1RhbA78r67XJcuK/2xD4aVYByd6PETVM1I7Bq0+LrECeEDqsTMcGHfAwc6ir69Lw0t/KVG/yHwbWyfwiYbkajfcQWYU669rD6ikY1B87assrqPDXneu65FxmoCYDMwBGRp8/w2gS7HSZmom5zA=
+    bucket: coveo-ndev-binaries
+    local-dir: bin
+    upload-dir: proda/StaticCDN/searchui/pr/$TRAVIS_BRANCH
+    acl: public_read
+    skip_cleanup: true
+    on:
+      repo: coveo/search-ui
+      all_branches: true
+      condition: "$IS_PULL_REQUEST_PUSH_BUILD == true"
```

Comment:
Deployment providers can't be run on a `continuous-integration/travis-ci/pr` build. There's probably a way to make that work, but `continuous-integration/travis-ci/pr` pull changes from `master` and their builds seem to be mostly for testing anyways.